docs/KUBERNETES.md

```
@@ -206,9 +213,23 @@ kubectl create secret generic mmrelay-credentials \
 
 #### Enable credentials Secret in deployment
 
-1. Uncomment the credentials Secret volume in `deployment.yaml`:
-   - In the `volumes` section, uncomment the credentials Secret volume
-   - In `spec.template.spec.containers[0].volumeMounts`, uncomment the credentials mount
+1. Add a credentials Secret volume to `deployment.yaml`:
+   - In the `volumes` section, add:
+     ```yaml
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)